src/expressions/if-expr.md

```diff
@@ -53,11 +53,11 @@ assert_eq!(y, "Bigger");
 
 An `if let` expression is semantically similar to an `if` expression but in
 place of a condition expression it expects the keyword `let` followed by a
-refutable pattern, an `=` and an expression. If the value of the expression on
-the right hand side of the `=` matches the pattern, the corresponding block
-will execute, otherwise flow proceeds to the following `else` block if it
-exists. Like `if` expressions, `if let` expressions have a value determined by
-the block that is evaluated.
+pattern, an `=` and an expression. If the value of the expression on the right
```

Comment:
```suggestion
pattern, an `=` and a scrutinee expression. If the value of the scrutinee
```
(scrutinee defined: https://github.com/rust-lang-nursery/reference/blob/95a0133d5998252c0eee616e7eb485a74a51de71/src/glossary.md#scrutinee ; you might want to link there..)